spec/controllers/families_controller_spec.rb

```diff
@@ -25,4 +25,41 @@
       end
     end
   end
+
+  describe "PUT /families/:id" do
+    context "When the family id exist" do
+      it "updates the family" do
+        family = create(:family)
+
+        put :update, params: {
+          id: family.id,
+          family: { last_name: "Dark Blue" },
+        }
+
+        expect(family.reload.last_name).to eq("Dark Blue")
+      end
+
+      it "sets an alert message" do
+        family = create(:family)
+
+        put :update, params: {
+          id: family.id,
+          family: { last_name: "Dark Blue" },
+        }
+
+        expect(flash[:notice])
+          .to match(
+            I18n.t("flash.actions.update.notice", resource_name: "Family")
+          )
+      end
+    end
+
+    context "When the family id doesn't exist" do
+      it "flashes an alert message and show the families list page" do
```

Comment:
A suggestion to change the descriptions:
```
context "when family does not exist" do
  it "raises a record not found error"
    #...
```

Reply:
Done in sha: ```5b22239```